src/Microsoft.DotNet.Build.Tasks/PackageFiles/scripts/vstsagent/CloneWithRetry.cmd

```diff
@@ -0,0 +1,35 @@
+@echo off
+
+if "%1" == ""   goto :usage
+if "%2" == ""   goto :usage
+goto :main
+
+:usage
+echo Usage: %~n0 [repository url] [destination folder]
+echo.
+echo Clones a git repo using the specified URL with depth 1 (for performance), 
+echo retrying up to 10x if there is a failure.
+goto :eof
+
+:main
+set REPO_URL=%1
+set FOLDER_NAME=%2
+set /a RETRY_COUNT=10
+set /a SLEEP_TIME=10
+
+IF NOT DEFINED GIT SET "GIT=%PROGRAMFILES(X86)%\Git\cmd\git.exe"
```

Comment:
Is GIT the env we use for finding it? Should we consider the version on the path?